pkg/services/accesscontrol/dualwrite/reconciler.go

```diff
@@ -88,6 +90,12 @@ func NewZanzanaReconciler(cfg *setting.Cfg, client zanzana.Client, store db.DB,
 				zanzanaCollector([]string{zanzana.RelationAssignee}),
 				client,
 			),
+			newResourceReconciler(
+				"anonymous role binding",
+				anonymousRoleBindingsCollector(cfg, store),
+				zanzanaCollector([]string{zanzana.RelationAssignee}),
+				client,
+			),
```

Comment:
I think we should only add this one if anonymous auth is enabled

Reply:
Good point